Source/WebKit/UIProcess/API/C/WKAPICast.h

```diff
@@ -250,6 +250,8 @@ inline WKProcessTerminationReason toAPI(ProcessTerminationReason reason)
     case ProcessTerminationReason::RequestedByNetworkProcess:
     case ProcessTerminationReason::RequestedByGPUProcess:
     case ProcessTerminationReason::Crash:
+    case ProcessTerminationReason::GPUProcessCrashedTooManyTimes:
+    case ProcessTerminationReason::ModelProcessCrashedTooManyTimes:
         return kWKProcessTerminationReasonCrash;
```

Comment:
Do we not prefer adding a new C-API type here?

```suggestion
        return kWKProcessTerminationReasonCrash;
    case ProcessTerminationReason::GPUProcessCrashedTooManyTimes:
    case ProcessTerminationReason::ModelProcessCrashedTooManyTimes:
        return kWKProcessTerminationReasonExceededSharedProcessCrashLimit;
```

Reply:
C API is deprecated as far as I know.